### ISSUE TITLE
PMB layout color mapping

### ISSUE THREAD

**Issue by pabloech** [PPiMapBuilder/PPiMapBuilder#68]
For v0.13 please eliminate the dark blue colors form the palette, the name of the nodes in protein and functions in black cannot being seen.

**Comment by gcornut** [comment]
Or maybe, if it's easier, we could turn the node text color to white on dark node color. That way we could maintain a large number of possible colors. What do you think Pierre? (since you are the one who created the palette)

**Comment by pidupuis** [comment]
Yes I was aware of this problem. When I developed the style I tried to apply a color mapping on label but I had some problem with the API so I temporary let it with black label.

However, I think that having two label colors (black and white) could be a little bit noisy. I also though about the possibility to use only lighter color (by taking only high RGB values for example).

Anyway I planned to do some tests in order to solve this.
